packages/grafana-ui/src/options/builder/valueFormatting.tsx

```diff
@@ -0,0 +1,46 @@
+import { OptionsWithValueFormatting } from '../models.gen';
+import { PanelOptionsEditorBuilder } from '@grafana/data';
+
+export enum VizValueFormattingMode {
+  Auto = 'auto',
+  Fixed = 'fixed',
+}
+export type VizValueFormattingOptions = {
+  size?: number;
+  mode: VizValueFormattingMode;
+};
+
+export function addValueFormattingOptions<T extends OptionsWithValueFormatting>(builder: PanelOptionsEditorBuilder<T>) {
+  builder
+    .addRadio({
+      path: 'valueFormatting.mode',
+      name: 'Value label size mode',
+      category: ['Value formatting'],
+      description: '',
+      defaultValue: VizValueFormattingMode.Auto,
+      settings: {
+        options: [
+          {
+            value: VizValueFormattingMode.Auto,
+            label: 'Auto',
+          },
+          {
+            value: VizValueFormattingMode.Fixed,
+            label: 'Fixed',
+          },
+        ],
+      },
+    })
```

Comment:
do we need this option? for the stat/bargauge/gauge we have a font size option, if you leave it empty it means Grafana will automatically handle it 

The option name (path), shoudl we align it with the naming for stat/bar gauge/ https://github.com/grafana/grafana/blob/main/public/app/plugins/panel/stat/types.ts#L138   (text.valueSize)

Reply:
@torkelo this is now unified between all visualizations using text size. Added to common options builders